### ISSUE TITLE
Wooshy isn't working for me on Firefox.

### ISSUE THREAD

**Issue by YousufSSyed** [godbout/Wooshy.blahblah#125]
What can be done to troubleshoot it?

**Comment by godbout** [comment]
which version of Firefox are you using? the normal edition and the developer edition (see https://github.com/godbout/Wooshy.docs/issues/58#issuecomment-1209288537) should be supported.

**Comment by YousufSSyed** [comment]
Regular Firefox, version 114.0.2.

**Comment by godbout** [comment]
all good here:

https://github.com/godbout/Wooshy.docs/assets/121373/89ead254-ee6e-4aa8-8a3b-bdf47b61cc48

can you check this Issue https://github.com/godbout/Wooshy.docs/issues/3 and see if turning ON some settings in Firefox help? thanks.

**Comment by YousufSSyed** [comment]
I skimmed through and I figured it out: its because I had manually disabled accessibility in `about:config`. Re-enabling it made it work. Thanks.

Btw, have you thought of adding an FAQ or troubleshooting section to the docs?

**Comment by godbout** [comment]
> I skimmed through and I figured it out: its because I had manually disabled accessibility in `about:config`. Re-enabling it made it work. Thanks.

great.

> Btw, have you thought of adding an FAQ or troubleshooting section to the docs?

sure. but i don't think it's the right time. i prefer one to one contact and fix people's issues the way we're doing it now. once i'll have too much volume then i'll add a FAQ. i'm trying to keep things to a minimum for the users. they just contact me and i'm here.

another reason is that Wooshy (and my other apps) deal with a very specific and niche part of macOS (the Accessibility) and there's gonna be issues with many different apps, etc. so i need to investigate the stuff on a per app basis. PLUS on a per app version basis, because they may break the AX between versions. so the FAQ would just be a massive shitshow i think, at least when it's related to issues like yours, for a specific app.

**Comment by okgolove** [comment]
Sorry for necroposting :)
For me Wooshy does not work with Firefox run on non main display. It finds elements, but wrong position

<img width="871" alt="image" src="https://github.com/godbout/Wooshy.docs/assets/36846182/ea60a848-d0df-4c4d-b184-f83066c9f9ec">

**Comment by godbout** [comment]
@okgolove only happens with Firefox right? i remember checking extensively and finding out that basically Firefox returns wrong data when using several monitors. pretty sure i've reported and talked about in some issues in Wooshy (or kV) but can't find it 🤔️

**Comment by okgolove** [comment]
@godbout yep, this is only about Firefox and only on secondary display.

**Comment by godbout** [comment]
@okgolove from that comment it seems that my tests at that time showed that Firefox does return wrong data: https://github.com/godbout/Wooshy.docs/issues/3#issuecomment-1200171416

seems also i hadn't reported that bug: https://bugzilla.mozilla.org/buglist.cgi?query_format=advanced&emailtype1=exact&emailreporter1=1&email1=guill.bout%40gmail.com&list_id=16963154

what i'm gonna do now is:
1. test again
2. try with other tools and see if the results are the same
3. if yes, file a bug to Mozilla

**Comment by godbout** [comment]
Firefox working now on multiple monitors, when displays don't have separate spaces:

https://github.com/user-attachments/assets/e3c75e15-5b42-4231-9717-1dd42763948c

**Comment by godbout** [comment]
and when displays have separate spaces:

https://github.com/user-attachments/assets/c5c1fca2-c09f-4c84-ad11-75256c0ddd06

Firefox people must have been back to work. closing. please reopen if you still have issues.

**Comment by godbout** [comment]
actually still breaking if the second monitor resolution is high enough. Firefox issue. will report again and as usual, thoughts and prayers.

https://github.com/user-attachments/assets/e1d717f0-1ded-4fce-bbf5-b02215a99bef

**Comment by godbout** [comment]
reported here: https://bugzilla.mozilla.org/show_bug.cgi?id=1949713

i would encourage everyone to do the same so that Mozilla pays attention to this.

**Comment by miiraheart** [comment]
Having the same issue on Zen, hopefully it gets fixed

**Comment by godbout** [comment]
> Having the same issue on Zen, hopefully it gets fixed

yep, Zen is based on Firefox. issue still open, no news from them.
